tests/wpt/meta/custom-elements/state/state-css-selector-nth-of.html.ini

```diff
@@ -1,10 +1,6 @@
 [state-css-selector-nth-of.html]
-  expected: ERROR
   [state selector has influence on nth-of when state is applied]
     expected: FAIL
 
-  [state selector only applies on given ident]
-    expected: NOTRUN
-
   [style is invalided on clear()]
-    expected: NOTRUN
+    expected: FAIL
```

Comment:
It seems that we incorrectly apply `:nth-child(2 of :state(foo))`. I'm not sure why these tests fail but I'm not entirely convinced that it's related to `:state` at all.